config/schedule.rb

```diff
@@ -3,19 +3,31 @@
 # Use this file to easily define all of your cron jobs.
 # Learn more: http://github.com/javan/whenever
 
+require 'config'
+
+Config.load_and_set_settings(Config.setting_files('config', 'production'))
+
 # these cron jobs all append to existing log files
 
+# These define jobs that checkin with Honeybadger.
+# If changing the schedule of one of these jobs, also update at https://app.honeybadger.io/projects/54415/check_ins
+job_type :rake, "cd :path && :environment_variable=:environment bundle exec rake :task --silent :output && curl 'https://api.honeybadger.io/v1/check_in/:check_in"
+job_type :runner_hb, "cd :path && bin/rails runner -e :environment ':task' && curl 'https://api.honeybadger.io/v1/check_in/:check_in' :output"
+
 # 11 am on the 1st of every month
+# If changing schedule, also change for HB checkin
 every :month, at: '11:00', roles: [:queue_populator] do
   set :output, standard: nil, error: 'log/m2c-err.log'
-  runner 'MoabStorageRoot.find_each(&:m2c_check!)'
+  set :check_in, Settings.honeybadger_checkins.moab_to_catalog
+  runner_hb 'MoabStorageRoot.find_each(&:m2c_check!)'
 end
 
 # 11 am on the 15th of every month - the 'whenever' syntax for this is awkward and needs an ignored month
 # for the day to get parsed, so just use raw cron syntax
 every '0 11 15 * *', roles: [:queue_populator] do
   set :output, standard: nil, error: 'log/c2m-err.log'
-  runner 'MoabStorageRoot.find_each(&:c2m_check!)'
+  set :check_in, Settings.honeybadger_checkins.catalog_to_moab
```

Comment:
add warning to ALL of these about changing HB checkin?  Or move warning to be seaparate from first job?